pkg/tiles/buildings/bulding.go

```diff
@@ -0,0 +1,20 @@
+package buildings
+
+type Bulding int64
+
+const (
+	NoneBuilding Bulding = iota
```

Comment:
Building in NoneBuilding is redundant and could be simplified to just None

Reply:
noned